include/seqan3/io/structure_file/format_vienna.hpp

```diff
@@ -129,9 +129,10 @@ class structure_file_format_vienna
               comment_type & SEQAN3_DOXYGEN_ONLY(comment),
               offset_type & SEQAN3_DOXYGEN_ONLY(offset))
     {
-        auto stream_view = view::subrange<decltype(std::istreambuf_iterator<char>{stream}),
-                                          decltype(std::istreambuf_iterator<char>{})>
-                           { std::istreambuf_iterator<char>{stream}, std::istreambuf_iterator<char>{} };
+        using stream_char_t = typename stream_type::char_type;
```

Comment:
```suggestion
        using stream_t = typename std::istreambuf_iterator<typename stream_type::char_type>;
```
make the lines below shorter and also line 259

Reply:
True! That's a really good idea.